### DISCUSSION TITLE
says I exceeded quota, but usage shows no quota used..

### DISCUSSION THREAD

**Discussion by MokshaK** [plandex-ai/plandex#116] [Q&A]
I loaded 23d_summary for context via plandex load;
then in plandex tell, I copy/pasted 23d_summary

I got following error as shown in screenshot, but when I checked usage page in OpenAI account, it seems like I have enough credit? I'm currently on the plus plan.  

please advise where I made mistake or advice.  thank you

<img width="725" alt="errors" src="https://github.com/plandex-ai/plandex/assets/2703167/3c4305c9-af4d-4f93-80b8-4962d50a1820">
<img width="1259" alt="usage" src="https://github.com/plandex-ai/plandex/assets/2703167/cd071e30-df65-4fd4-8c9b-305aabe54e02">

[23d_plan.txt](https://github.com/plandex-ai/plandex/files/15282974/23d_plan.txt)
[23d_summary.txt](https://github.com/plandex-ai/plandex/files/15282975/23d_summary.txt)

**Comment by danenania** [comment]
Hey @MokshaK, it can take some time for the OpenAI usage stats to be updated.
